_pages/clients/nostr-emitter.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Nostr-Emitter
+permalink: /nostr-emitter/
+web: https://cmdruid.github.io/nostr-emitter/examples/nostr-chat/
+github: cmdruid/nostr-emitter
+instances:
+- name: Nostr-Emitter
+  url: https://cmdruid.github.io/nostr-emitter/examples/nostr-chat/
+  comment: Instance managed by @cmdruid
+tags:
+- social
```

Comment:
This is not a "social client". It looks more like a tech demo. At some point we will have to expand on all the tags but for now we are exploring. To my understanding:

* client: it's not a relay or documentation but a software that talks to relays
* social: it allows you to create a profile and follow others and share **kind 1** text notes. "Twitter-like"
* chat: it allows to join potentially public chat rooms. "Telegram-like"
* dm: it allows Encrypted Direct Message chat using **kind 4** events.
* chess: jester for example is for chess
* game: jester is also a game

```suggestion
- chat
```

Reply:
Noted on this.